README.md

```diff
@@ -24,13 +24,13 @@ You can compile the binary by doing `script/setup` and `script/compile`. Golang
   OR
   docker-gc (-command=ttl) (-interval=INTERVAL_IN_SECONDS) (-images_ttl=DURATION) (-containers_ttl=DURATION) for continuous cleanup based on image/container TTL
   OR
-  docker-gc (-command=diskspace) (-interval=INTERVAL_IN_SECONDS) (-high_disk_space_threshold=PERCENTAGE) (-low_disk_space_threshold=PERCENTAGE) for continuous cleanup based on used disk space
+  docker-gc (-command=diskspace) (-interval=INTERVAL_IN_SECONDS) (-high_disk_space_threshold=PERCENTAGE) (-low_disk_space_threshold=PERCENTAGE) (-containers_ttl=DURATION) for continuous cleanup based on used disk space
```

Comment:
Yeah () was optional, but this was totally random not a convention. I changed to what you proposed